src/main/java/olivero/tasks/Task.java

```diff
@@ -0,0 +1,65 @@
+package olivero.tasks;
+
+/**
+ * Represents an abstract task to be extended from.
+ */
+public abstract class Task {
+    private final String description;
+    private boolean isDone;
+
+    /**
+     * Constructs a default implementation of a subclass of {@code Task} object.
+     *
+     * @param description The task description.
+     * @param isDone The task completion status.
+     */
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Sets the completion status of the task to the provided boolean value.
+     *
+     * @param done The provided boolean value representing task completion status.
+     */
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    /**
+     * Returns true if this task is completed and false otherwise.
+     *
+     * @return Task completion status.
+     */
+    public boolean isDone() {
```

Comment:
```suggestion
    public boolean getIsDone() {
```

Don't think this follows getter naming conventions, should be getIsDone()!

Reply:
Hi, since this is a boolean method, it has been named to sound like a boolean based on the coding conventions